test/e2e/README.md

```diff
@@ -1,6 +1,59 @@
-# End-to-End Tests
+# Kubestellar end to end testing
+
+This e2e tests can test either (a) the  local copy of the repo or (b) the release identified in the kubestellar PostCreateHook (which will be the last release created, regardless of quality, except for that brief time when it identifies the release about to be made). Testing the local copy is the default behavior; to test the release identified in the PostCreateHook, pass `--released` on the command line of `run-test.sh`.
```

Comment:
This directory does not contain one test. It contains a generic test runner and two subdirectories that each has a distinct test.

Reply:
ok, I understand your comment better now. Just updated the content to address it.